lmvachon/bad.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<title>Lauren's Bad Code Example</title>
+
+	<link rel="stylesheet" href="/css/style.css" type="text/css" media="all" />
+</head>
+
+<body>
+
+	<header>
+		<nav>
+			<a href="#">Home
+			<a href="#">About
+			<a href="#">Contact
+		</nav>
+	</header>
+	<section>
+		<article>
+			<header>
+				<h2>Article title</h2>
```

Comment:
Is `h2` the appropriate level of header given that it's the only header on the page so far? Do pages like this usually get an `h1` for the page title? If so, should that be added before the 'article' proper begins?